# Lock Status
bool isRollLocked
bool isYawLocked
bool isPitchLocked 
bool isDepthLocked

# PID
int64 kp_roll
int64 ki_roll
int64 kd_roll
int64 target_stp_roll
int64 kp_pitch
int64 ki_pitch
int64 kd_pitch
int64 target_stp_pitch
int64 kp_yaw
int64 ki_yaw
int64 kd_yaw
int64 target_stp_yaw
int64 kp_depth
int64 ki_depth
int64 kd_depth
int64 target_stp_depth

#kontrol joy
string leftX
string leftY
string rightX
string rightY
string R1
string R2
string L1
string L2

#GY Calibrate
bool isGYCalibrate

